src/Unirota.Application/Commands/SolicitarEntrada/SolicitarEntradaGrupoCommand.cs

```diff
@@ -0,0 +1,11 @@
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Remover using desnecessário (validar se é usado ou não, se for usado, vai dar erro quando vc remover :))

Reply:
Using é usado, portanto, necessário